usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -50,13 +41,8 @@ def post(
             )
             return self.get_download_response(file_name=filename)
 
-        if validator_type is not None:
-            validator = validator_type(request.data)
-            json_request = validator.json_request
-        else:
-            json_request = request_type.value["validate_func"](request.data)
-            json_request["request_type"] = request_type.value["name"]
-        json_request = order_nested_object(json_request)
+        validator = validator_type(request.data)
```

Comment:
`validator_type` is an optional parameter, will removing the check that it exists cause problems if it's not provided.

Reply:
Realistically it should not because we are deciding what validator type to pass back not the user, but I can add a check and throw an error to make sure it is clear. It is only optional if you provide a "special" request type where we pre-generate it (right now only disaster download).

Reply:
I think add some error handling here would be useful for future developers